fuel-chain-config/src/config/chain.rs

```diff
@@ -30,6 +30,8 @@ use super::{
 pub const LOCAL_TESTNET: &str = "local_testnet";
 pub const TESTNET_INITIAL_BALANCE: u64 = 10_000_000;
 
+// TODO: Remove not consensus/network fields from `ChainConfig` or create a new config only
```

Comment:
Which fields are you referring to here?

Reply:
It was about `block_production` and `chain_name`. But maybe `chain_name` may live here, but I can't say that is a "required" field for us=D

The `block_production` describes how to behave and is more related to `fuel-block-producer` because of `fuel_poa_coordinator::Trigger::Never`(In the network the only part of nodes uses `Never`). Maybe we need another type to describe production rules for `ChainConfig`.

Reply:
In the case of networks with multiple producers, they should have the same block production trigger config imo.

Reply:
Yes, but if you are sentry node, your `ChainConfig` will be different=)

Reply:
Ok but lets say in a multi-producer network, we want to use this field to configure the initial set of validator keys. It shouldn't change whether you are in sentry mode or not. To me that seems like production behavior could be based on whether you actually have a local private key configured that matches the configured validator set?

Reply:
Yes, but my concern is that right now the type used for `block_production` is not right. All nodes in the network should have the same `ChainConfig` regardless of the role of the node. This type fits very well for `ProducerConfig`, but for `ChainConfig` we need another